website/docs/command-reference/catalog.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "catalog"
+---
+This document is a work in progress.
```

Comment:
Does it make sense to have a document without any content?

Reply:
I think it would only frustrate the readers. I'll remove those.